packages/paste-website/src/components/shortcodes/component-header/index.tsx

```diff
@@ -2,160 +2,123 @@ import * as React from 'react';
 import {Helmet} from 'react-helmet';
 import {Anchor} from '@twilio-paste/anchor';
 import {Box} from '@twilio-paste/box';
+import {Stack} from '@twilio-paste/stack';
 import {Text} from '@twilio-paste/text';
 import {Heading} from '@twilio-paste/heading';
-import {Breadcrumb, BreadcrumbItem} from '../../breadcrumb';
+import {useTheme} from '@twilio-paste/theme';
 import {PackageStatusLegend} from '../package-status-legend';
-import {SidebarCategoryRoutes, STORYBOOK_DOMAIN} from '../../../constants';
-import {P, InlineCode} from '../../Typography';
-import {getHumanizedNameFromPackageName, getOpengraphServiceUrl} from '../../../utils/RouteUtils';
-import {sentenceCase} from '../../../utils/SentenceCase';
-import type {PackageStatusObject} from '../../../utils/types';
+import {STORYBOOK_DOMAIN} from '../../../constants';
+import type {SidebarCategoryRoutes} from '../../../constants';
```

Comment:
It can't be a type... It's used a bunch of different places as a const. If I import it as a const in this file, I get an eslint error bc it is only used as type.... so I think for now this is the best solution.

Reply:
Hmm... When I switch to that I still need to do `import type {SidebarCategoryRoutes} from '../../../constants';`

I'm going to leave it as is for now.